src/Cache/ArrayResult.php

```diff
@@ -16,19 +16,15 @@
  */
 final class ArrayResult implements Result
 {
-    private int $columnCount = 0;
-    private int $num         = 0;
+    private readonly int $columnCount;
```

Comment:
`readonly` properties must not have a default. The old default was moved to the constructor logic.